src/api-serverless/src/nextgen/nextgen.routes.ts

```diff
@@ -452,6 +452,69 @@ router.get(
   }
 );
 
+router.get(
+  `/collections/:id/ultimate_trait_set`,
+  async function (req: any, res: any, next: any) {
+    const id: number = parseInt(req.params.id);
+    const traits = req.query.trait;
+
+    if (isNaN(id)) {
+      return res.status(404).send({});
+    }
+
+    if (!traits) {
+      return res.status(400).send({});
+    }
+
+    logger.info(
+      `[FETCHING ULTIMATE TRAIT SETS FOR COLLECTION ID ${id} TRAITS ${traits}]`
+    );
```

Comment:
Not necessary. We log all requests full URL's down anyway (including query params). And the HTTP response codes